src/Month.js

```diff
@@ -216,7 +215,7 @@ class MonthView extends React.Component {
     let { localizer, components } = this.props
     let first = row[0]
     let last = row[row.length - 1]
-    let HeaderComponent = components.header || Header
```

Comment:
You should leave header from lib as default if header not came from outside

Reply:
updated